web/README.md

```diff
@@ -23,7 +23,22 @@ $ yarn start
 
 For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
 
-## for developer
+### Environment Variables
```

Comment:
How about creating `docs` dir in `web/` (like in `simulator/`) and then creating `env-variables.md` in it?

Reply:
It also looks Good:+1:
That would be the same as the simulator's one.

However, this file name is not to my liking. (Because we usually use the word `env` mean as `environment-variable`, it sounds like `environment-variable-variables.md`)
I want to rename it to either of the following.
- `env-vers.md`
- `environment-variables.md`

WDYT?

Reply:
(Of course, the simulator's one, too)

Reply:
Either one looks fine for me👌